openfga/v1/openfga_service.proto

```diff
@@ -821,6 +833,9 @@ message ListObjectsRequest {
   // Additional request context that will be used to evaluate any ABAC conditions encountered
   // in the query evaluation.
   google.protobuf.Struct context = 7;
+
+  // Controls the consistency preference for this request. Default is MINIMIZE_LATENCY.
```

Comment:
I don't think this is correct. Default is minimize latency. But we can mention that currently unspecified falls back to minimize latency?

If we need MINIMIZE_LATENCY to be the default it has to be 0 I think

Reply:
Updated the docs to be clearer here. Behaviorally, UNSPECIFIED will be treated as MINIMIZE_LATENCY, but you are correct the actual default value is UNSPECIFIED.